src/commands/CmdNews.cpp

```diff
@@ -546,23 +606,27 @@ int CmdNews::execute (std::string& output)
       items.end ()
     );
 
-  // Print release notes
   Color bold = Color ("bold");
-  std::cout << bold.colorize (format (
-    "\n"
-    "==========================================\n"
-    "Taskwarrior {1} {2} Release highlights\n"
-    "==========================================\n",
-    version,
-    (full_summary ? "All" : (major_items ? "Major" : "Minor"))
-   ));
-
-  for (unsigned short i=0; i < items.size (); i++) {
-    std::cout << format ("\n({1}/{2}) ", i+1, items.size ());
-    items[i].render ();
+  if (items.empty ()) {
+    std::cout << bold.colorize ("You are up to date!\n");
+  } else {
+    // Print release notes
+    std::cout << bold.colorize (format (
+      "\n"
+      "============================================\n"
+      "Taskwarrior {1} - {2} Release highlights\n"
```

Comment:
I don't really have a better suggestion but I find this format confusing. Dashes are often used within version numbers which is why I don't typically see them used to denote ranges. I tried a few formulations using version specifier syntax (e.g., `>{1}, <={2}`) but couldn't find one that was a clear improvement.

You didn't introduce this but I also find it odd that "highlights" is not capitalized.

Reply:
I could replace it with "through".

Reply:
Only if you think that's better. Neither that nor what you currently have seem strictly accurate because the `news_version` release notes are actually not included, it's everything after `news_version`, right?

Another idea: `{1} -> {2}`.

Reply:
That's true! But I suspect users will forgive some loose use of range expressions :)